src/main/resources/spring.gradle

```diff
@@ -23,7 +25,12 @@ dependencies {
     // json-formatted log file
     implementation "net.logstash.logback:logstash-logback-encoder:${findProperty('logstash-logback-encoder.version') ?: '7.2'}"
     // support for json-patch is optional
-    implementation "com.github.java-json-tools:json-patch:${findProperty('json_patch.version') ?: '1.13'}"
+    implementation "com.github.java-json-tools:json-patch:${findProperty('json-patch.version') ?: '1.13'}"
+    // support object mapping with map-struct
+    implementation "org.mapstruct:mapstruct:${MAPSTRUCT_VERSION}"
+    annotationProcessor "org.mapstruct:mapstruct-processor:${MAPSTRUCT_VERSION}"
+    annotationProcessor "org.projectlombok:lombok-mapstruct-binding:${findProperty('lombok-mapstruct-binding.version') ?: '0.2.0'}"
```

Comment:
I think it will be good to specify default component model as compile time option to the processor in order to use spring dependency injection directly to create mapper instances 
```
compileJava {
    options.compilerArgs += [
        '-Amapstruct.defaultComponentModel=spring'
    ]
}
```
this will allow as to define the mapper without explicitly using component model param in the @Mapper annotation

Reply:
I would prefer to have this annotation. I don't see a reason to skip it.